config/config.inc.php

```diff
@@ -221,7 +221,13 @@
 if (isset($cookie->id_lang) && $cookie->id_lang) {
     $language = new Language($cookie->id_lang);
 }
-if (!isset($language) || !Validate::isLoadedObject($language) || !$language->isAssociatedToShop()) {
+
+$isNotValidLanguage = !isset($language) || !Validate::isLoadedObject($language);
+// `true` if language is defined from multishop or backoffice session
+$isLanguageDefinedFromCustomSession = (isset($language) && $language->isAssociatedToShop()) || isset($employee);
+
+$useDefaultLanguage = $isNotValidLanguage || !$isLanguageDefinedFromCustomSession;
+if ( $useDefaultLanguage ) {
```

Comment:
I'm surprised we don't have a code styling rule for this, but as we usually don't put spaces for simple `if`, I suggest you to remove them.

```suggestion
if ($useDefaultLanguage) {
```

Reply:
@atomiix ,

I took some time to search if I could configure something to help (cause I'll never be able to be attentive to this alone ^^') and I found that :
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5709
and
https://packagist.org/packages/superdj/spaces-in-parentheses-php-cs-fixer

But it seems this fixer isn't compatible with version we used ^^'

Should be had the link to some roadmap ? Where can I find this ?

Reply:
Actually, the rule is used, but the config of `php-cs-fixer` omit the `config/` folder.

Reply:
Oh ! I'll check this then ^^